msgs/src/main/scala/com/thenewmotion/ocpi/msgs/v2_1/Sessions.scala

```diff
@@ -34,9 +34,9 @@ object Sessions {
   }
 
   case class Session(
-    sessionId: SessionId,
-    startDateTime: ZonedDateTime,
-    endDateTime: ZonedDateTime,
+    id: SessionId,
+    startDatetime: ZonedDateTime,
```

Comment:
Hmm, don't like this serialisation-induced renaming, to be honest.

Reply:
Better than having to declare all the field names IMHO...